packages/lisk-transactions/src/utils/validation/schema.ts

```diff
@@ -121,6 +184,9 @@ export const signatureTransaction = {
 		source: { $ref: 'lisk/base-transaction' },
 		with: {
 			properties: {
+				amount: {
+					format: 'nonTransferAmount',
```

Comment:
why validateNonTransferAmount? The amount should be equal to zero. nonTransferAmount will return false for negative values.